Bio/Nexus/Nexus.py

```diff
@@ -299,7 +299,7 @@ def _seqmatrix2strmatrix(matrix):
 
 
 def _compact4nexus(orig_list):
-    """Transform [1 2 3 5 6 7 8 12 15 18 20] (baseindex 0, used in the Nexus class)
+    r"""Transform [1 2 3 5 6 7 8 12 15 18 20] (baseindex 0, used in the Nexus class)
```

Comment:
@etal - do you know what this docstring is intending to show the user with slashes in Nexus? The current rendering via epydoc looks wrong to me http://biopython.org/DIST/docs/api/Bio.Nexus.Nexus-module.html#_compact4nexus (likely a side effect of switching to assuming reStructuredText in the past?).